src/test/java/net/testudobank/tests/MvcControllerIntegTest.java

```diff
@@ -275,18 +273,16 @@ public void testWithdrawOverdraftOverage() throws SQLException, ScriptException
    LocalDateTime timeWhenWithdrawRequestSent = MvcControllerIntegTestHelpers.fetchCurrentTimeAsLocalDateTimeNoMilliseconds();
    System.out.println("Timestamp when withdraw request sent: " + timeWhenWithdrawRequestSent);
 
-
-   //verify that customer1's Overdraft balance is equal to remaining withdraw amount with interest applied
-   int CUSTOMER1_ORIGINAL_BALANCE_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER1_BALANCE);
-   int CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER1_AMOUNT_TO_WITHDRAW);
-   int CUSTOMER1_EXPECTED_OVERDRAFT_BALANCE_BEFORE_INTEREST_IN_PENNIES = CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES - CUSTOMER1_ORIGINAL_BALANCE_IN_PENNIES;
-   int CUSTOMER1_EXPECTED_OVERDRAFT_BALANCE_AFTER_INTEREST_IN_PENNIES = (int)(CUSTOMER1_EXPECTED_OVERDRAFT_BALANCE_BEFORE_INTEREST_IN_PENNIES * MvcController.INTEREST_RATE);
-   System.out.println("Expected Overdraft Balance in pennies: " + CUSTOMER1_EXPECTED_OVERDRAFT_BALANCE_AFTER_INTEREST_IN_PENNIES);
-
-   
    //Check the response when the withdraw request is submitted. This should return the user back to the home screen due to an invalid request
    String responsePage = controller.submitWithdraw(customer1WithdrawFormInputs);
    assertEquals("welcome", responsePage);
+   
+   //verify that customer1's Overdraft balance is equal to remaining withdraw amount with interest applied
+   int CUSTOMER1_ORIGINAL_BALANCE_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER1_BALANCE);
+   int CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES = MvcControllerIntegTestHelpers.convertDollarsToPennies(CUSTOMER1_AMOUNT_TO_WITHDRAW);
+   int CUSTOMER1_OVERDRAFT_BALANCE_INCREASE_BEFORE_INTEREST_IN_PENNIES = CUSTOMER1_AMOUNT_TO_WITHDRAW_IN_PENNIES - CUSTOMER1_ORIGINAL_BALANCE_IN_PENNIES;
+   int CUSTOMER1_OVERDRAFT_BALANCE_INCREASE_AFTER_INTEREST_IN_PENNIES = (int)(CUSTOMER1_OVERDRAFT_BALANCE_INCREASE_BEFORE_INTEREST_IN_PENNIES * MvcController.INTEREST_RATE);
+   System.out.println("Expected Overdraft Balance in pennies: " + CUSTOMER1_OVERDRAFT_BALANCE_INCREASE_AFTER_INTEREST_IN_PENNIES);
```

Comment:
Just remove this print statement. You use the word `"Expected"` again in the print statement. Also it should be clear from the method's name and its summary comment that the customer's balance should stay the same, so this extra documentation is not required to send to the output.

If this integ test were to crash, the terminal will display this test's name, which should indicate to the user what this integ test cast is trying to do.

I am thinking about adding a Debug Logging suite to our codebase so that we can be more lenient with outputting stuff to the terminal while testing.